src/AdminSettings.vue

```diff
@@ -110,6 +111,25 @@
 				</p>
 			</li>
 		</ul>
+		<ul class="password-generator__settings-list">
```

Comment:
Will be good to put a printscreen with this rendered at description of this PR to have a best explaining about your contribution.